lib/screens/explore_screens/regions_screens/join_region/interactor/viewmodels/join_region_bloc.dart

```diff
@@ -15,16 +18,25 @@ class JoinRegionBloc extends Bloc<JoinRegionEvent, JoinRegionState> {
   JoinRegionBloc() : super(JoinRegionState.initial()) {
     on<OnLoadRegions>(_onLoadRegions);
     on<OnUpdateMapLocation>(_onUpdateMapLocations);
-    on<OnRegionResultSelected>(_onRegionResultSelected);
   }
 
   Future<void> _onLoadRegions(OnLoadRegions event, Emitter<JoinRegionState> emit) async {
     emit(state.copyWith(pageState: PageState.loading));
-    final result = await GetRegionsUseCase().run();
+    final result = await GetRegionUseCase().run(settingsStorage.accountName);
```

Comment:
if you grab the `settingsStorage.accountName` inside the user case, you can use a NoInputUseCase 
instead of InputUseCase
Not a blocker tho. just pointing this out.

Reply:
happens I did it thinking that we could use any account not only the one that is in the storage.

Reply:
yeah, thats cool